src/docs/previews/dropdown-menu/main/tailwind/index.svelte

```diff
@@ -144,6 +144,7 @@
 		@apply data-[highlighted]:ring-magnum-400 data-[highlighted]:ring-offset-2 !important;
 		@apply p-0 text-sm font-medium  data-[highlighted]:outline-none;
 	}
+	/* isn't this duplicated */
```

Comment:
Thanks for the feedback, but please remove the comments so that they don't appear in docs :) (all TW examples will be deleted after we add the CSS ones)